src/command.coffee

```diff
@@ -185,7 +186,7 @@ compileScript = (file, input, base = null) ->
     else
       compiled = CoffeeScript.compile t.input, t.options
       t.output = compiled
-      if o.map
+      if o.map and not o['inline-map']
```

Comment:
@DylanPiercey What do you mean by switching to babel? Moving to ES6? Or - did you make it work with coffeescript? I'd love to see this merged as it would be really really beneficial for us! :+1: